server/apps/demo/Dockerfile

```diff
@@ -2,7 +2,7 @@
 #
 # VERSION	1.0
 
-FROM apache/james:jpa-3.6.0
+FROM apache/james:jpa-latest
```

Comment:
the `latest` tag does not exist in the docker hub.

Reply:
But DEV building from the source will have it and we can switch the tag upon the release .